sirconfig.h

```diff
@@ -90,7 +90,11 @@
  *
  * @remark Only applies if ::SIRO_NOPID or ::SIRO_NOTID are not set.
  */
-# define SIR_PIDFORMAT "%d"
+# if defined(__MINGW64__)
+#  define SIR_PIDFORMAT "%lld"
```

Comment:
I think it's better to cast to int rather than hacking the macro, tbh. Why did they make PIDs long long? They have 2^64-1 processes?

Reply:
No idea.  It's a long long int on 64-bit MinGW and just a int on 32-bit MinGW.  The upstream is aware that this is discrepancy compared to native Windows compilers but does not wish to make any changes.  

I see QEMU solves this exact problem with a cast - https://lists.nongnu.org/archive/html/qemu-devel/2015-09/msg06804.html

so I'll do the cast.